src/windows/system.rs

```diff
@@ -435,6 +435,14 @@ impl SystemExt for System {
     }
 
     fn long_os_version(&self) -> Option<String> {
+        if self.is_windows_eleven() {
+            return get_reg_string_value(
+                HKEY_LOCAL_MACHINE,
+                "SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion",
+                "ProductName",
+            )
+            .map(|product_name| product_name.replace("10", "11"));
```

Comment:
This is definitely not great. For example it can return `10 (191045)`. With this `replace`, the second `10` would be replaced. It's still a bit hacky but I think something like this:

```rust
produce_name.replace("10 (", "11 (")
```

would work better

Reply:
Thank you very much for your time. 🙇 

Function `long_os_version` mentioned above replace "ProductName" like `Windows 10 XXXXX`.

I changed codes to replace the string "Windows 10 " with "Windows 11 ".
I confirmed that this replacement covers all of the possible ProductName.

- Windows 10 Home
- Windows 10 Pro
- Windows 10 Education
- Windows 10 Enterprise
- Windows 10 S